.github/workflows/dast-release.yml

```
@@ -0,0 +1,291 @@
+    environment: production
+    permissions:
+      contents: read
+      id-token: write
+      attestations: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: backend
+            image_repo: app-stayhealthy-backend
+            image_digest: ${{ inputs.image_digest_backend }}
```

actionlint (1.7.12)

[error] property "image_digest_backend" is not defined in object type {image_digest: string; image_name: string; registry: string; registry_username: string; sbom-predicate-type: string; sbom-repository: string}

(expression)